databases/db.js

```diff
@@ -79,6 +81,30 @@ inventory.belongsTo(Sticker);
 Event.hasOne(inventory);
 inventory.belongsTo(Event);
 
+//Relaciones entre usuarios y stickers para el almacen
+User.belongsToMany(Sticker, { 
+  through: Warehouse,
+});
+Sticker.belongsToMany(User, { 
+  through: Warehouse
+});
+User.hasMany(Warehouse);
+Warehouse.belongsTo(User);
+Sticker.hasMany(Warehouse);
+Warehouse.belongsTo(Sticker);
+
+Event.hasOne(Warehouse);
```

Comment:
```suggestion
Event.hasMany(Warehouse);
```
Entiendo que, los eventos tienen VARIAS "warehouses" que realmente no son warehouses como tal si no la relacion entre un sticker ("jugador en este contexto"), un usuario y un evento

Reply:
Cñ, xd

Reply:
Si o no xD? explicate chamo

Si la hiciste dale a "Resolver" si no, explicame por que para caernos a pingasos o para yo aprender xd